### DISCUSSION TITLE
Wide and Deep Recommender

### DISCUSSION THREAD

**Discussion by scott-alvis** [recommenders-team/recommenders#1963] [Q&A]
Hello,

I'm hoping someone could shed some light on the behavior of the wide and deep recommender. I'm using the Azure Machine Learning Studio Wide and Deep Recommender module. I've built several different models, from the most basic model (user ratings only) to a model with user ratings, user features, and item features. For a given user no matter what data I pass into the model, for scoring, I get the same exact recommendations. I've varied the items and ratings being passed. I've varied the user features being passed. I've varied the item features being passed. No matter what I send into the model for a given user I get the same exact recommendations.

Is this the expected behavior of the model?

Any help or insight would be greatly appreciated.

Thank you,

Scott

**Comment by miguelgfierro** [comment]
In this repo there is an implementation of the W&D model. Not sure how the model of Azure ml studio works.
